Rakefile

```diff
@@ -5,6 +5,17 @@ rescue LoadError
 end
 require 'rubygems/package_task'
 require 'rspec/core/rake_task'
+
+# Fix for NoMethodError: undefined method `last_comment'
+# Deprecated in Rake 12.x
```

Comment:
The alternative to this workaround would be to pin `rake` to `~> 11`.

Reply:
For context, failing build: https://travis-ci.org/stitchfix/resqutils/jobs/632012636

And the source of the workaround: https://stackoverflow.com/questions/35893584/nomethoderror-undefined-method-last-comment-after-upgrading-to-rake-11